contracts/CoveragePool.sol

```diff
@@ -23,24 +23,29 @@ import "@openzeppelin/contracts/math/SafeMath.sol";
 
 /// @title CoveragePool
 /// @notice A contract that manages a single asset pool. Handles approving and
-/// unapproving of risk managers and allows them to seize funds from the asset
-/// pool if they are approved.
-/// @dev Coverge pool contract is owned by the governance. Coverage pool is the
-/// owner of the asset pool contract.
+///         unapproving of risk managers and allows them to seize funds from the
+///         asset pool if they are approved.
+/// @dev Coverage pool contract is owned by the governance. Coverage pool is the
+///      owner of the asset pool contract.
 contract CoveragePool is Ownable {
     using SafeMath for uint256;
 
     AssetPool public assetPool;
     IERC20 public collateralToken;
 
+    bool public firstRiskManagerApproved = false;
+
     // Currently approved risk managers
     mapping(address => bool) public approvedRiskManagers;
     // Timestamps of risk managers whose approvals have been initiated
     mapping(address => uint256) public riskManagerApprovalTimestamps;
     // Timestamps of risk managers whose unapprovals have been initiated
     mapping(address => uint256) public riskManagerUnapprovalTimestamps;
 
-    /// @notice Throws if called by a risk manager that has not been approved
+    event RiskManagerApprovalStarted(address indexed riskManager);
```

Comment:
Can we add `uint256 timestamp` to `RiskManagerApprovalStarted` event? We use this pattern for governance-related events in risk manager. I'd also suggest to not index `riskManager` address on events. It is not needed in this case - those events will be emitted rarely and risk manager address should always be the only and unique value.

Reply:
... Since we are here can we add events for the unapproval process as well? 😬 Same about get remaining time function.